reqs/requirements-dev-test.txt

```diff
@@ -2,3 +2,4 @@
 --requirement requirements-dev-lite.txt
 --requirement requirements-dev-contrib.txt
 --requirement requirements-dev-api-docs-test.txt
+--requirement requirements-dev-snowflake.txt
```

Comment:
```suggestion
```
I don't think we do this as this is causing all the tests to install the snowflake deps.